packages/app-staking/src/Actions/Account/SetSessionAccount.tsx

```diff
@@ -44,61 +45,61 @@ class Key extends React.PureComponent<Props, State> {
 
     return (
       <Modal
-        className='staking--Stash'
+        className='staking--SetSessionAccount'
         dimmer='inverted'
         open
         size='small'
       >
         {this.renderContent()}
         <Modal.Actions>
-        <Button.Group>
-          <Button
-            isNegative
-            onClick={onClose}
-            label={t('Cancel')}
-          />
-          <Button.Or />
-          <TxButton
-            accountId={accountId}
-            isDisabled={!sessionId || !!sessionError}
-            isPrimary
-            label={t('Set Session Key')}
-            onClick={onClose}
-            params={[sessionId]}
-            tx='session.setKey'
-          />
-        </Button.Group>
-      </Modal.Actions>
+          <Button.Group>
```

Comment:
I have tried to get the linter to complain about these - maybe it will go better once we are on eslint...